src/librustc_resolve/lib.rs

```diff
@@ -1471,6 +1477,7 @@ impl<'a> Resolver<'a> {
                 Scope::MacroRules(..) => true,
                 Scope::CrateRoot => true,
                 Scope::Module(..) => true,
+                Scope::RegisteredAttrs => true,
```

Comment:
Or-patterns in various arms would be good here.

Reply:
I thinks it's better to list scopes in the same order here than merge arms.